controllers/virtualmachinepublishrequest/virtualmachinepublishrequest_controller.go

```diff
@@ -320,10 +322,12 @@ func (r *Reconciler) publishVirtualMachine(ctx *pkgctx.VirtualMachinePublishRequ
 			ctx.Logger.Error(err, "update VirtualMachinePublishRequest status failed")
 			return err
 		}
+		vm := ctx.VM
+		contentLibrary := ctx.ContentLibrary
 
 		go func() {
 			actID := getPublishRequestActID(vmPublishReq)
-			itemID, pubErr := r.VMProvider.PublishVirtualMachine(ctx, ctx.VM, vmPublishReq, ctx.ContentLibrary, actID)
+			itemID, pubErr := r.VMProvider.PublishVirtualMachine(ctx, vm, vmPublishReq, contentLibrary, actID)
```

Comment:
I don't see these vars used anywhere else besides this function call. Was this change intentional?